lib/mercury/cps/seq.rb

```diff
@@ -13,13 +12,13 @@ def m
         @m ||= Cps.identity # we need an initial Cps to chain onto
       end
 
-      def chain(proc=nil, &block)
+      def chain(proc = nil, &block)
```

Comment:
The team had a hipchat discussion on this a while ago and decided that the space-less version is more readable 9 out of 10 times.